docker/pwsh-exchange/Dockerfile

```
@@ -0,0 +1,5 @@
+
+FROM demisto/powershell-ubuntu:7.0.3.12001
+
+RUN pwsh -c "Set-PSRepository -Name PSGallery -InstallationPolicy Trusted -ErrorAction Stop"
+RUN pwsh -c "Install-Module -ErrorAction Stop -Scope AllUsers -Name ExchangeOnlineManagement -RequiredVersion 2.0.4-Preview2 -AllowPrerelease -Force"
```

```suggestion
RUN pwsh -c "Install-Module -Name ExchangeOnlineManagement -RequiredVersion 2.0.4-Preview2 -AllowPrerelease -Force"
```
1. No need to stop on error, It will still have the right exit code (Docker image build will be failed).
2. Scope AllUsers  - Its the default value ( For an elevated PowerShell session, Scope defaults to AllUsers)

---

@GalRabin let's leave this. We add this automatically in our `create_new_docker_image.py` script. It is better to have this explicit.

---

Ok